src/wizards/tokenSwapDealWizard/components/stageButtons/stageButtons.ts

```diff
@@ -8,6 +8,13 @@ import "./stageButtons.scss";
 export class stageButtons {
   @bindable wizardManager: any;
   @bindable showSubmit = false;
+  validating = false;
 
-  constructor(public wizardService: WizardService) {}
+  constructor(public wizardService: WizardService) {
+  }
+
+  async proceed() {
+    this.validating = true;
```

Comment:
```suggestion
    this.isLoading= true;
```
Because, to me, it seems like you are only controlling the `is-loading` in the `<pbutton>` here